administrator/language/en-GB/plg_behaviour_compat6.ini

```
@@ -4,4 +4,10 @@
 ; Note : All ini files need to be saved as UTF-8
 
 PLG_BEHAVIOUR_COMPAT6="Behaviour - Backward Compatibility 6"
-PLG_COMPAT6_XML_DESCRIPTION="This plugin should be enabled before updating to Joomla 6."
+PLG_COMPAT6_FIELD_CLASSES_ALIASES_DESCRIPTION="Add class aliases for classes which have been renamed or moved to a namespace."
+PLG_COMPAT6_FIELD_CLASSES_ALIASES_LABEL="Classes Aliases"
+PLG_COMPAT6_FIELD_LEGACY_CLASSES_DESCRIPTION="Activate this option if your extension requires classes deprecated for the current version of Joomla. This includes those removed classes from the folder /plugins/behavior/compat6/classes/."
```

Not 100% sure but is this a correct alternative

```suggestion
PLG_COMPAT6_FIELD_LEGACY_CLASSES_DESCRIPTION="Activate this option if your extension requires previously deprecated classes removed from the current version of Joomla. These removed classes can be found in the folder /plugins/behavior/compat6/classes/."
```